model/uzivatel-slucovani.php

```diff
@@ -5,6 +5,27 @@
  */
 class UzivatelSlucovani {
 
+  /**
+   * @return pole dvojic [tabulka, sloupec] odkazující na id_uzivatele
+   */
+  private function odkazujiciTabulky() {
+    $odkazy = dbQuery('
+      SELECT DISTINCT table_name, column_name
```

Comment:
Zkoušel jsem ten DISTINCT a lokálně mi to s ním i bez něj dává 9 výsledků bez duplicit. Zkus to pls u sebe (můžeš mít jinou verzi databáze nebo jakýkoli podobný důvod) jestli se ti to podaří zreprodukovat, a pokud to byl jen nějaký dočasný bug a teď už to duplicity nehází, DISTINCT můžeš vyhodit. Pokud by tam u tebe duplicity furt byly, zkus mi popsat postup a rád bych to nasimuloval u sebe a ještě dál ladil.